src/ctap/test_helpers/env.rs

```diff
@@ -0,0 +1,49 @@
+use crate::clock::CtapInstant;
+use crate::ctap::command::{
+    AuthenticatorAttestationMaterial, AuthenticatorConfigParameters,
+    AuthenticatorVendorConfigureParameters, Command,
+};
+use crate::ctap::data_formats::ConfigSubCommand;
+use crate::ctap::status_code::Ctap2StatusCode;
+use crate::ctap::{key_material, Channel, CtapState};
+use crate::env::Env;
+
+// In tests where we define a dummy user-presence check that immediately returns, the channel
+// ID is irrelevant, so we pass this (dummy but valid) value.
+const DUMMY_CHANNEL: Channel = Channel::MainHid([0x12, 0x34, 0x56, 0x78]);
```

Comment:
Regarding the `vendor_hid` question, I guess you want to have 2 dummy channels, one main and one vendor. Depending whether `vendor_hid` feature is enabled, you want to use the vendor dummy channel for vendor commands.

Reply:
done